inspirehep/modules/workflows/static/js/inspire_workflows_ui/holdingpen/holdingpen.filters.js

```diff
@@ -78,7 +78,10 @@
         }
       }
 
-      return categories;
+      var uniqueCategories = categories.filter(function (value, index, self) {
+        return self.indexOf(value) === index;
+      });
```

Comment:
yes could be done better, but as you said, array is like to be small, max 5-6 I think. So didn't want to complicate the code for a little performance improvement :)